### PR TITLE
[consensus] Add smoke test for consensus testing it's fault tolerance.

### PR DESCRIPTION
**Pull request by ikabiljo**
### Test Plan

---
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2133)

### CODE REVIEW

File: consensus/src/epoch_manager.rs [hunk 1]
```diff
@@ -711,6 +713,9 @@ impl EpochManager {
         &self,
         request: IncomingBlockRetrievalRequest,
     ) -> anyhow::Result<()> {
+        fail_point!("consensus::process::any", |_| {
```
Comment (bchocho) [inline consensus/src/epoch_manager.rs:716]: `consensus::process::any` is already used above

File: consensus/src/network.rs [hunk 2]
```diff
@@ -92,6 +92,9 @@ impl NetworkSender {
         from: Author,
         timeout: Duration,
     ) -> anyhow::Result<BlockRetrievalResponse> {
+        fail_point!("consensus::send::any", |_| {
```
Comment (bchocho) [inline consensus/src/network.rs:95]: Should be another fail_point name?

**Review by bchocho — COMMENTED** [review]
Not sure if this is ready for review? Found a few repeated failpoint names. I can look more into this if it's ready.

**Comment by ikabiljo** [comment]
chatted offline, I intentionally put the same name everywhere to be able to turn off all processing or all sending with a single failpoint.

**Pull request closed without merging**